go.mod

```diff
@@ -2,48 +2,62 @@ module github.com/calyptia/cli
 
 go 1.18
 
-replace github.com/calyptia/cli/k8s => ./k8s
-
 require (
 	code.cloudfoundry.org/bytefmt v0.0.0-20211005130812-5bb3c17173e5
+	github.com/aws/aws-sdk-go-v2 v1.16.7
+	github.com/aws/aws-sdk-go-v2/config v1.15.14
+	github.com/aws/aws-sdk-go-v2/service/ec2 v1.49.1
 	github.com/calyptia/api v0.3.9
-	github.com/calyptia/cli/k8s v0.0.0-00010101000000-000000000000
+	github.com/calyptia/cli/k8s v0.0.0-20220716151659-d687bbfd5c34
+	github.com/calyptia/core-images-index/go-index v0.0.0-20220722082249-a242d67500e9
 	github.com/calyptia/go-bubble-table v0.2.1
 	github.com/charmbracelet/bubbletea v0.22.0
 	github.com/charmbracelet/lipgloss v0.5.0
 	github.com/hako/durafmt v0.0.0-20210608085754-5c1018a4e16b
 	github.com/joho/godotenv v1.4.0
 	github.com/matryer/moq v0.2.7
 	github.com/pkg/errors v0.9.1
+	github.com/sethvargo/go-retry v0.2.3
 	github.com/spf13/cobra v1.5.0
 	github.com/zalando/go-keyring v0.2.1
 	golang.org/x/sync v0.0.0-20220601150217-0de741cfad7f
 	golang.org/x/term v0.0.0-20220526004731-065cf7ba2467
 	gopkg.in/yaml.v2 v2.4.0
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
-	k8s.io/client-go v0.24.2
+	k8s.io/api v0.24.3
+	k8s.io/apimachinery v0.24.3
+	k8s.io/client-go v0.24.3
 )
 
 require (
 	github.com/alessio/shellescape v1.4.1 // indirect
-	github.com/charmbracelet/bubbles v0.12.0 // indirect
+	github.com/aws/aws-sdk-go-v2/credentials v1.12.9 // indirect
+	github.com/aws/aws-sdk-go-v2/feature/ec2/imds v1.12.8 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/configsources v1.1.14 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/endpoints/v2 v2.4.8 // indirect
+	github.com/aws/aws-sdk-go-v2/internal/ini v1.3.15 // indirect
+	github.com/aws/aws-sdk-go-v2/service/internal/presigned-url v1.9.8 // indirect
+	github.com/aws/aws-sdk-go-v2/service/sso v1.11.12 // indirect
+	github.com/aws/aws-sdk-go-v2/service/sts v1.16.9 // indirect
+	github.com/aws/smithy-go v1.12.0 // indirect
+	github.com/charmbracelet/bubbles v0.11.0 // indirect
```

Comment:
I see that quite a few dependencies were downgraded. What happeneded here? 🤔 

Try deleting `go.sum` and removing all depenencies from `go.mod` (leave just the module name, go version and the replace directive). Then:
```
go get -u -t ./...
go mod tidy
```